bench/src/lib.rs

```diff
@@ -221,6 +253,18 @@ pub struct Opt {
     /// Disable packet encryption/decryption
     #[clap(long)]
     no_protection: bool,
+    /// Simulate network in-memory, instead of using the network stack
+    #[clap(long)]
+    simulate_network: bool,
+    /// Simulated link delay (one way), in milliseconds
+    #[clap(long, default_value = "0")]
```

Comment:
`requires("simulate_network")` here and below? I think that's the syntax, at least.